ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -6,15 +6,15 @@
 import UIKit
 
 class TabBarItemView: UIControl, TokenizedControlInternal {
-    public func overrideTokens(_ tokens: TabBarTokens?) -> Self {
+    public func overrideTokens(_ tokens: TabBarItemTokens?) -> Self {
         overrideTokens = tokens
         return self
     }
     let item: TabBarItem
 
-    var defaultTokens: TabBarTokens = .init()
-    var tokens: TabBarTokens = .init()
-    var overrideTokens: TabBarTokens? {
+    var defaultTokens: TabBarItemTokens = .init()
+    var tokens: TabBarItemTokens = .init()
```

Comment:
I think the decision on private subcontrols is that they would not be tokenized and the tokens that define their appearance would be defined in the container public control (TabBarView in this case).

It seems to be a more lightweight approach than having each private control support the tokens infrastructure.

Reply:
I've made a quick change to demonstrate why I feel this is a problematic approach, especially for this control. To me, it it ends up being a lot more cumbersome than a nicely encapsulated Tokens class that is reset automatically on token and theme changes.